### ISSUE TITLE
Bad left-side bearings of text uc italic glyphs

### ISSUE THREAD

**Issue by Amphiboly** [khaledhosny/libertinus#561]
Some uc italic glyphs have too much space on the left. In particular, the _T_ and _W_, and the bold **_N_** suffer from this, and other glyphs (_C_, _G_ _O_, _Q_, and _U_) suffer to a lesser extent. This results in uneven spacing in text, and more importantly, ugly left margins when one of these glyphs is at the left of the line. This is with release _7.051._

![image](https://github.com/user-attachments/assets/05764b3f-9b9c-403b-9ca0-1ff4541dcefd)

I would expect to see consistent spacing without for all of these, including their bold variants, without the extra padding.

A similar issue was previously addressed for math fonts (#252, #256).

**Comment by alerque** [comment]
I agree. Besides T and W, O and Q look pretty off to me from your preview. Possible a few others to lesser degrees.

Contributions are welcome. I probably won't be able to jump on this soon but all the tooling is open source and this should be a relatively easy fix if somebody wants to jump in. Besides the LSB, one would need to make sure any existing kerning values don't get messed up when the change is made. I would start with the most egregious letters and work backwards. The hardest decision will probably be when to stop touching more letters. The fewer changes the better as this will cause reflows for people, but its still a worthwhile fix I think.
